packages/web/src/pages/Honeypots/VaultDetailsPage/Sections/VaultScopeSection/InScopeSection/InScopeSection.tsx

```
@@ -215,6 +216,17 @@ export const InScopeSection = ({ vault }: InScopeSectionProps) => {
         </Alert>
       )}
 
+      {/* LoC */}
+      {vault.description["project-metadata"].loc && (
+        <>
+          <h4 className="section-subtitle mb-4">
+            <HashtagIcon className="icon" />
+            <span>{t("linesOfCode")}:</span>
+            <span>{vault.description["project-metadata"].loc}</span>
+          </h4>
+        </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)